Source/Core/DiscIO/VolumeTri.h

```diff
@@ -0,0 +1,108 @@
+// Copyright 2008 Dolphin Emulator Project
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#include <map>
+#include <memory>
+#include <optional>
+#include <string>
+#include <vector>
+
+#include "Common/CommonTypes.h"
+#include "Common/Lazy.h"
+#include "DiscIO/Filesystem.h"
+#include "DiscIO/Volume.h"
+#include "DiscIO/VolumeDisc.h"
+
+namespace DiscIO
+{
+class BlobReader;
+enum class BlobType;
+enum class Country;
+class FileSystem;
+enum class Language;
+enum class Region;
+enum class Platform;
+
+class VolumeTri : public VolumeDisc
```

Comment:
Could probably mark this `final`

Reply:
Mark the VolumeTri class final?

Reply:
Yes

Reply:
Why only VolumeTri? Wouldn't VolumeGC be eligible as well? Or any of the other classes in DiscIO?

Reply:
Yes they would.  `VolumeTri` is new code, the others would need to happen in a cleanup review most likely.